test/check-machines-filesystems

```diff
@@ -60,8 +60,12 @@ class TestMachinesFilesystems(machineslib.VirtualMachinesCase):
 
         b.set_input_text("#vm-subVmTest1-filesystems-modal-mountTag", "")
         b.set_input_text("#vm-subVmTest1-filesystems-modal-source-group input", "/tmp")
-        b.wait_visible("#vm-subVmTest1-filesystems-modal-source")
+        # In some hosts with high latency, the path dropdown list needs some time to load, the check will fail if it is too quick.
+        # Thus, add more check to ensure that the loading is finished
+        b.wait_in_text("#vm-subVmTest1-filesystems-modal-source", "/tmp")
```

Comment:
This is what I'm sceptical about: `set_input_text` already waits until the input has the expected value, so this is entirely redundant. Either that existing wait in testlib.py isn't enough -- then we should fix it there for all cases. Or it is redundant, then this is nothing more than a hidden "sleep(0.1)", and the real fix is to wait for something else.

Reply:
Yeah, set_input_text already waits until the input has the expected value. But, there will be a dropdown list for file path prompt as the input there is a FileAutoComplete. This line aims to make sure the input text is shown in the dropdown list. We don't check the dropdown list in set_input_text. And it seems that, if I don't add this line, the dropdown list is still there after b.click("#vm-subVmTest1-filesystems-modal-source button:nth-child(1)"). Here is also a picture if I remove this line.
![TestMachinesFilesystems-testBasicSystemConnection-dropdown-list](https://github.com/user-attachments/assets/5774ce00-b35c-4009-97ce-287e3b6b3ea2)

Reply:
Sorry for the late answer, I've been sick for two weeks :cry: Yes, I understand the "wait for /tmp to be in dropdown" approach. But the wait_in_text() is not specific enough for that -- it covers the *whole* input group, i.e. including the input line. Doesn't it also catch *that* `/tmp` there? I.e. my question is, this selector needs to be more specific to the select, with `.pf-v5-c-menu` or similar.

Reply:
I checked with cockpit-machines-316-1.fc40.noarch. #vm-subVmTest1-filesystems-modal-source doesn't include the input line, and the input line is actual in another div, and the id of the input line is "pf-select-toggle-id-26-select-typeahead", which is automatically generated, I think. Are there some changes about the input line and the expand list in the newest vision?

Reply:
No, there are no changes. The main thing is that the selectors are not understandable, and could use some comments what they actually refer to.

Thanks for the explanation! Yes, please rebase, and let's get this in.